### PR TITLE
NDS-175 Filters component

### PR DESCRIPTION
**Pull request by wa-rren-dev**
https://nicedigital.atlassian.net/browse/NDS-175

### CODE REVIEW

**Review by ediblecode — DISMISSED** [review]

File: components/nds-filters/src/FilterPanel.js [hunk 1]
```diff
@@ -28,33 +29,56 @@ export class FilterPanel extends Component {
...
-					aria-hidden={!isExpanded}
-				>
-					{children}
-					{!canUseDOM && (
-						<button type="submit" className="btn filter-panel__submit">
-							Apply filters
+			<form onSubmit={onSubmit} {...rest} {...fallback} ref={innerRef}>
+				<div className={classnames(["filter-panel", className])}>
+					<HeadingLevel className="filter-panel__heading">
+						<button
```
[the remark this answers is not in the document]
Comment (wa-rren-dev) [reply components/nds-filters/src/FilterPanel.js:60]: @johndavey72 are you good to roll this into what you're doing today?

File: components/nds-filters/src/FilterSummary.js [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+	...rest
+}) {
+	const HeadingLevel = "h" + headingLevel;
+
+	return (
+		<div className={classnames("filter-summary", className)} {...rest}>
+			<div className="filter-summary__count">
+				<HeadingLevel className="h5 mv--0">{children}</HeadingLevel>
+			</div>
+			{sorting.length && (
```
Comment (ediblecode) [inline components/nds-filters/src/FilterSummary.js:23]: @johndavey72 - another for your PR today pls: this is rendering a `0` if there are no items:
image

**Review by ediblecode — CHANGES_REQUESTED** [review]

File: components/nds-filters/src/FilterPanel.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (johndavey72) [reply components/nds-filters/src/FilterPanel.js:60]: yes I will take a look

File: components/nds-filters/src/FilterSummary.js (hunk 2 above)
Comment (johndavey72) [reply components/nds-filters/src/FilterSummary.js:23]: leave it with me

File: components/nds-filters/src/FilterSummary.js [hunk 3]
```diff
@@ -0,0 +1,190 @@
+import React from "react";
+import PropTypes, { elementType } from "prop-types";
+import classnames from "classnames";
+import { Tag } from "@nice-digital/nds-tag";
+import RemoveIcon from "@nice-digital/icons/lib/Remove";
+import "./../scss/filter-summary.scss";
+import validate from "validate-npm-package-name";
```
Comment (ediblecode) [inline components/nds-filters/src/FilterSummary.js:7]: What is this doing here? It seemed to cause issues when I installed the latest alpha version?

File: components/nds-filters/src/FilterSummary.js [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+					destination,
+					method,
+					elementType,
+					onClick,
+					className = ""
+				}) => {
+					const ElementType = defineElementType(onClick, elementType);
+					const props = {
+						className: classnames(["tag__remove", className]),
+						"aria-label": `Sort by ${label}`,
```
Comment (ediblecode) [inline components/nds-filters/src/FilterSummary.js:73]: This should say Remove not Sort by @johndavey72 can you fix pls?
```suggestion
						"aria-label": `Remove ${label} filter`,
```

**Review by ediblecode — CHANGES_REQUESTED** [review]

**Pull request closed without merging**